spec/filters/mutate_spec.rb

```diff
@@ -85,6 +93,28 @@
     end
   end
 
+  describe "case handling of multibyte unicode strings will only change ASCII" do
+    config <<-CONFIG
+      filter {
+        mutate {
+          lowercase => ["lowerme"]
+          uppercase => ["upperme"]
+        }
+      }
+    CONFIG
+
+    event = {
+      "lowerme" => [ "АБВГД\0MMM", "こにちわ", "XyZółć"],
+      "upperme" => [ "аБвгд\0mmm", "こにちわ", "xYzółć"],
+    }
```

Comment:
I suggest we also include "classic" french upper/lower accented chars like:
ùûüÿàâæçéèêëïîôœ
ÙÛÜŸÀÂÆÇÉÈÊËÏÎÔŒ